snarker/src/consensus/consensus_state.rs

```diff
@@ -37,6 +37,29 @@ impl ConsensusShortRangeForkDecision {
     }
 }
 
+#[derive(derive_more::From, Serialize, Deserialize, Debug, Clone)]
+pub enum ConsensusLongRangeForkDecisionIgnoreReason {}
+
+#[derive(derive_more::From, Serialize, Deserialize, Debug, Clone)]
+pub enum ConsensusLongRangeForkResolutionKind {
+    SubWindowDensity,
+    ChainLength,
+    Vrf,
+    StateHash,
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub enum ConsensusLongRangeForkDecision {
+    Keep(ConsensusLongRangeForkResolutionKind),
```

Comment:
Can we rename this for consistency?
`Keep` -> `Ignore`
`Take` -> `Use`

Reply:
Keep/take is a terminology from Mina implementation, like here:
https://github.com/MinaProtocol/mina/blob/472c82cded7362594be3681cba5d41862477d817/src/lib/consensus/proof_of_stake.ml#L2959

I thought that using it is a good idea.

Reply:
I agree about consistency. We can choose either Mina terminology or Ignore/Use.

Also I would get rid of negated enum for the short range fork.

Reply:
Cool. Let's use mina terminology then for short range fork also.